src/vue/backend/src/components/APIKey.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <idx-form-group class="idx-wp-needs-validation" :class="{'idx-wp-was-validated': success|error}" novalidate>
```

Comment:
There are two uses of class here. In order to use our prefix, you can make these both `customClass`